Tools/machines/perlmutter-nersc/install_cpu_dependencies.sh

```diff
@@ -109,6 +109,45 @@ CXX=$(which CC) CXXFLAGS="-DLAPACK_FORTRAN_ADD_" cmake -S $HOME/src/lapackpp -B
 cmake --build ${build_dir}/lapackpp-pm-cpu-build --target install --parallel 16
 rm -rf ${build_dir}/lapackpp-pm-cpu-build
 
+# heFFTe
+if [ -d $HOME/src/heffte ]
+then
+  cd $HOME/src/heffte
+  git fetch --prune
+  git checkout v2.4.0
+  git pull
```

Comment:
This does not work. 
What about doing this instead: `git pull origin master` ?

Reply:
Ooops, same version was in the GPU version. We need to remove the
```suggestion
  git pull
```

Reply:
Fix shipping via #5006